tutorial/chapter/3/index.md

```diff
@@ -125,32 +115,51 @@ function(x, y) {
 }
 ```
 
-JavaScript's usual relaxed rules around argument count can lead to extremely
-subtle and hard to detect bugs. It's potentially more flexible the JavaScript
-way, but much more error-prone.
+JavaScript's usual relaxed rules around argument count can lead to extremely subtle and hard to detect bugs. It's potentially more flexible the JavaScript way, but much more error-prone.
+
+Sometimes APIs are much easier if they can have varying parameter counts, so it is still possible in Squiggle. Using `...` in front of your last parameter causes it so collect all of the extra parameters in an array.
```

Comment:
I think the phrasing of this sentence could be improved. I’d replace “easier” with “simpler” and “it is still possible” with “variadic functions are still possible” to be more clear.